### DISCUSSION TITLE
[Projects Beta] Show issue id in table

### DISCUSSION THREAD

**Discussion by mobile-sergey** [community/community#8131] [Projects and Issues] [answered]
We use issue id(number showing after hash sign) to name brunch in git. 
But in [Projects Beta] no way to show issue id. Only if we open issue we can see issue id - it's not comfortable

Update: Duplicate #5904
Fixed in `Q4 2021 – Oct-Dec`[GitHub public roadmap (Beta)](https://github.com/orgs/github/projects/4247/views/1)

**Comment by Katsute** [comment, marked as answer]
https://github.com/github/feedback/discussions/5904

**Reply by mobile-sergey** [reply]
Thanks. Good news that this task in the [Daily experience improvements for Projects #283](https://github.com/github/roadmap/issues/283)
which in the [GitHub public roadmap (Beta)](https://github.com/orgs/github/projects/4247/views/1) `Q4 2021 – Oct-Dec` 😀

**Reply by CalvinWilkinson** [reply]
Yippy!!  Glad to know that this is coming!!

**Comment by kdarras** [comment]
I could not find the info about being able to display issue id fields in the new projects in the link above, and the current projects experience still does not allow it, even when trying to add new fields:
![image](https://user-images.githubusercontent.com/7696198/186061325-67fb229f-e647-4642-843c-1396af2e26bd.png)
did I miss something? This would be extremely useful, because referring to unique ids/primary keys is essential for collaboration.

**Reply by CalvinWilkinson** [reply]
I absolutely agree with this.

**Reply by mobile-sergey** [reply]
Yes, I don't see it. But may be this only in test version now? And this would shown in future...
